nodes/rotation/wiggle_euler.py

```diff
@@ -0,0 +1,34 @@
+import bpy
+from bpy.props import *
+from math import pi
+from mathutils import Euler
+from ... events import propertyChanged
+from ... base_types.node import AnimationNode
+from ... algorithms.perlin_noise import perlinNoise
+
+class EulerWiggleNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_EulerWiggleNode"
+    bl_label = "Euler Wiggle"
+
+    nodeSeed = IntProperty(update = propertyChanged)
+
+    def create(self):
+        self.inputs.new("an_FloatSocket", "Seed", "seed")
+        self.inputs.new("an_FloatSocket", "Evolution", "evolution")
+        self.inputs.new("an_EulerSocket", "Amplitude", "amplitude").value = [pi / 6, pi / 6, pi / 6]
```

Comment:
sorry, just get the idea. Maybe its even better to use https://docs.python.org/3/library/math.html#math.radians here..